### PR TITLE
Improvements to the ALPIDE chip simulation

### PR DESCRIPTION
**Pull request by dr4kan**
This PR is to merge several changes to the ALPIDE chip simulation which include:
- (bug fix) adding a protection against unphysical values of ACS 
- dropping the use of TF1 and TF2
- adding the generation of noise in the simulation

### CODE REVIEW

**Comment by matthiasrichter** [comment]
Hi @iouribelikov, the problem with testMessageFormat should be fixed with PR #403, sorry about that.

**Comment by iouribelikov** [comment]
@matthiasrichter   Thank you very much, Matthias.  Apparently, now everything works fine !

File: Detectors/ITSMFT/common/simulation/src/SimulationAlpide.cxx [hunk 1]
```diff
@@ -129,21 +143,21 @@ SimulationAlpide::generateClusters(const SegmentationPixel *seg, DigitContainer
     Double_t beta = std::min(0.99999, trackP4.Beta());
     Double_t bgamma = beta / sqrt(1 - pow(beta, 2));
```
Comment (iouribelikov) [inline Detectors/ITSMFT/common/simulation/src/SimulationAlpide.cxx]: @dr4kan  Hi Davide.  Could you please quickly commit the changes requested by Sandro ?
Essentially:  replace "pow(x,2)" with "x*x",  and "sqrt" with "TMath::Sqrt"
Then, we would quickly merge this PR.

Thanks a lot,
Iouri.

**Comment by dr4kan** [comment]
@iouribelikov Hi Iouri I made the changes. Cheers, Davide.

**Comment by sawenzel** [comment]
@dr4kan : Thanks for the update. I am fixing a few minor things directly online and will squash/merge when the build succeeds.

**Comment by iouribelikov** [comment]
@dr4kan @sawenzel  Thank you both very much.

**Pull request merged**